package.json

```diff
@@ -7,24 +7,22 @@
     "node": ">=13.2.0"
   },
   "scripts": {
-    "start": "react-app-rewired start",
-    "start:debug": "node --inspect-brk node_modules/react-app-rewired/scripts/start.js",
-    "start:firefox": "react-app-rewired start --firefox",
-    "start:android": "react-app-rewired start --firefox-android",
-    "start:chromium": "react-app-rewired start --chromium",
-    "start:gecko": "react-app-rewired start --firefox-gecko",
-    "start:ios": "react-app-rewired start --wk-webview",
-    "build": "react-app-rewired build",
-    "build:base": "react-app-rewired build",
-    "build:debugging": "node --inspect-brk node_modules/react-app-rewired/scripts/build.js",
-    "build:gecko": "react-app-rewired build --firefox-gecko",
-    "build:firefox": "react-app-rewired build --firefox",
-    "build:chromium": "react-app-rewired build --chromium",
-    "build:ios": "react-app-rewired build --wk-webview",
+    "start": "webpack-dev-server",
+    "start:firefox": "webpack-dev-server --firefox",
+    "start:android": "webpack-dev-server --firefox-android",
+    "start:chromium": "webpack-dev-server --chromium",
+    "start:gecko": "webpack-dev-server --firefox-gecko",
+    "start:ios": "webpack-dev-server --wk-webview",
+    "build": "webpack --mode production",
+    "build:base": "webpack --mode production",
+    "build:gecko": "webpack --mode production --firefox-gecko",
+    "build:firefox": "webpack --mode production --firefox",
+    "build:chromium": "webpack --mode production --chromium",
+    "build:ios": "webpack --mode production --wk-webview",
     "test": "jest --watch",
     "test:ci": "jest --ci --collectCoverage=true --reporters=default --reporters=jest-junit",
     "storybook": "start-storybook -p 9009 -s public --quiet",
-    "storybook-build": "build-storybook -s public --quiet",
+    "build-storybook": "build-storybook -s public --quiet",
```

Comment:
`build:storybook`, uniform name style?

Reply:
the storybook cli doesn't recognize the name build:storybook. (though it isn't big deal)

Reply:
hmmmmmm, cursed